v4/export/writer_util.go

```diff
@@ -2,16 +2,74 @@ package export
 
 import (
 	"bufio"
+	"bytes"
+	"context"
 	"fmt"
 	"io"
 	"os"
 	"strings"
 	"sync"
+	"unsafe"
 
 	"github.com/pingcap/dumpling/v4/log"
 	"go.uber.org/zap"
 )
 
+const lengthLimit = 1048576
+
+func bytes2str(b []byte) string {
```

Comment:
Add a FIXME to get rid of this function in the future. We could specialize `write()` to accept `[]byte`, there's really no need to use unsafety.

Reply:
I have deleted `byte2str` and write directly